components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -25,4 +25,9 @@ governing permissions and limitations under the License.
   --spectrum-line-height-large: 1.7;
   --spectrum-line-height-medium: 1.5;
   --spectrum-line-height-small: 1.3;
+
+  --spectrum-checkbox-checkmark-size-small: 10px;
+  --spectrum-checkbox-checkmark-size-medium: 10px;
+  --spectrum-checkbox-checkmark-size-large: 10px;
+  --spectrum-checkbox-checkmark-size-extra-large: 12px;
```

Comment:
I can move them. I put them here because there weren't tokens for them and I had to hardcoded the values. Is the reason they should not be here because they are specific to the checkbox component?

Reply:
Yes. I think we should try to avoid putting component-specific things in here if possible, and keep these more "global" or general if possible.